### DISCUSSION TITLE
In my Local dev environment when i enable search in the table and when i try to enter search query the table vanishes.

### DISCUSSION THREAD

**Discussion by Ticket-Master** [filamentphp/filament#6693] [Help]
Hi, 

In my local dev the table search is making the table disappear totally. 
But the same thing works flawless in production. 

I tried deleting the filament views from resources->vendor and also tried deleting the vendor folder totally .

I deleted the whole directory and cloned my production repo on my local dev but still it is like that. 
![tablevanishes](https://github.com/filamentphp/filament/assets/77227724/522272e6-6628-4917-b934-f0bf8e9c9326)

What may be wrong?

**Comment by danharrin** [comment]
Please run `php artisan filament:upgrade`. If that doesn't fix it, try reproducing the issue in a separate app. If you can reproduce it with a simple example, open an issue.
